### ISSUE TITLE
[principle of operation converting] how to know how onnx optimize the softmax(or a pytorch specific operation)

### ISSUE THREAD

**Issue by songtaoshi** [onnx/tutorials#217]
### Question
**principle of softmax operation converting**    

Hi, I am new to onnx.   
When I convert my pytorch model to onnx, I find it divided into some operations.   
I am curious about the principle of converting an operation.   
Really hope to see some explanations considering both in **hardware & algorithm** perspective.  

### Further information
- principle of a specific operation converting, where could find the docs or explanation

**Comment by wenbingl** [comment]
Is torch.onnx doc enough for your question? 
https://pytorch.org/docs/stable/onnx.html
